docs/report.md

```diff
@@ -35,7 +35,16 @@ Here comes a description of our domain model.
 ## How to make _Chirp!_ work locally
 
 ## How to run test suite locally
-
+The test suite of Chirp is divided in two test-projects. 'Chirp.Infrastructure.Tests' and 'Chirp.Web.Test'.
+In order to run all test suites of the Chirp application, you simply need to be located in the root directory (Chirp) and run the following command:
+```
+dotnet test
+```
+First this should run the UI Tests located in the 'Chirp.Web.Test' project, and should return a prompt with 3 tests passed.
+After that is will run the integrationtests from the 'Chirp.Infrastructure.Tests' project, which should return a promt with 26 tests passed.
+
+As mentioned the 'Chirp.Web.Test' prject holds userinterface tests made using playwright. Upon review of the test class in this project it will be clear that there are more tests than the 3 being run. This is because we have five UI tests where we use a github 'test account' to test logged in features, theese have been a great help during development. The problem is that in order for them to run on a new computer, the github 'test account' will have to verify the new device with an email code. Therefore we have chosen to comment theese test out, in order for the reader to be able to run our test suite.
```

Comment:
```suggestion
As mentioned the 'Chirp.Web.Test' project holds user interface tests made using playwright. Upon review of the test class in this project it will be clear that there are more tests than the 3 being run. This is because we have five UI tests where we use a Github 'test account' to test logged in features, these have been a great help during development. The problem is that in order for them to run on a new computer, the Github 'test account' will have to verify the new device with an email code. Therefore we have chosen to comment these tests out, in order for the reader to be able to run our test suite.
```